app/femr/ui/views/history/indexEncounter.scala.html

```diff
@@ -177,15 +177,32 @@
                             <span class="value">@outputStringOrNA(viewModelPharmacy.getProblems.get(problemIterator - 1))</span></p>
                     }
 
-                    @for(prescription <- viewModelPharmacy.getPrescriptions){
-                        <p>
-                            <span class="infoLabel">Dispensed Medication:</span>
-                            @prescription.getAmount @outputStringOrNA(prescription.getName) (@outputStringOrNA(prescription.getMedicationForm).toString().trim)
-                            @for(activeDrug <- prescription.getMedicationActiveDrugs) {
-                                <span class='activeDrug'> @activeDrug.getName @activeDrug.getValue@activeDrug.getUnit </span>
-                            }
+                    @* AJ Saclayan Prescribed Medications *@
+                    @if(viewModelPharmacy.getOriginalMedications.size != null){
```

Comment:
AJ,

This is causing a warning from Java during compilation. The null check should exist to make sure you can call '.size()' on the original medications list. If 'viewModelPharmacy.getOriginalMedications()' is null, then you will get an error if you try to call .size().

The warning should go away if you remove .size().


This is the warning I get:
[warn] app/femr/ui/views/history/indexEncounter.scala.html:181: comparing values of types Int and Null using `!=' will always yield true
[warn]                     @if(viewModelPharmacy.getOriginalMedications.size != null){
[warn]                                                                       ^
[warn] one warning found